css-animationworklet/vision.md

```diff
@@ -0,0 +1,133 @@
+# Animation Worklet Vision
+***for rich interactive effects on the Web Platform***
+
+
+## Problem and Motivation
+
+**Fact**: Fact: It is difficult to build smooth rich interactive effects on the web.
+
+**Why do we care?**
+Silky smooth rich interactive user interfaces are now a [basic user expectation][performance] on
+modern computing platforms. For web platform to remain competitive it should be capable of high
+fidelity, smooth, responsive UI.
+
+
+**Where is the difficulty?**
+Three key aspects of rich interactive effects are: smoothness, responsiveness to input (a.k.a. R & A
+of [RAILS model][rails]) and their rich interaction model.
+
+The two main methods for creating animations on the web fall short in at least one of these aspects:
+
+- CSS (Web) Animations: Aimed at supporting stateless declarative time-driven effects. The
+  expressiveness is sufficient for common time-based effects. The resulting animation can be  smooth
+  if authors limit themselves to cheap-to-update properties<sup>[1](#footnote1)</sup>. However, it
```

Comment:
It's probably sufficient to move the "if authors limit themselves to cheap-to-update properties" to the footnote and just say the resulting animation can be smooth<sub>[1], especially since there is no difference with animation worklet in this regard.